pkg/ebpf/c/tracee.bpf.c

```diff
@@ -4388,6 +4388,8 @@ int uprobe_seq_ops_trigger(struct pt_regs *ctx)
     if (!init_program_data(&p, ctx))
         return 0;
 
+    p.event->context.matched_scopes = 0xFFFFFFFFFFFFFFFF;
```

Comment:
Please add to all triggered events.

Reply:
Missed that one. Will do it in another PR.